kiwi/system/uri.py

```diff
@@ -210,15 +219,29 @@ def _obs_distribution(self, name):
             )
         return obs_distribution
 
-    def _suse_buildservice_path(self, name):
+    def _suse_buildservice_path(self, name, urischeme, fragment=None):
         """
         Special to openSUSE buildservice. If the buildservice builds
         the image it arranges the repos for each build in a special
         environment, the so called build worker.
         """
-        return self._local_path(
-            '/usr/src/packages/SOURCES/repos/' + name
-        )
+        bs_source_dir = '/usr/src/packages/SOURCES'
+        if self.repo_type == 'container':
+            if urischeme == 'obsrepositories':
+                local_path = os.sep.join(
+                    [bs_source_dir, 'containers/_obsrepositories', name]
+                )
+            else:
+                local_path = os.sep.join(
+                    [bs_source_dir, 'containers', name]
+                )
+            if fragment:
+                local_path = ''.join([local_path, '#', fragment])
```

Comment:
The buildservice people told me they will provide it with the name as set in the uri. How they do the mapping to the full name is in their responsibility. So we we should be fine